### ISSUE TITLE
Please update Noto TTF

### ISSUE THREAD

**Issue by philippedostert** [osmc/osmc#604]
Hi,

I was researching why Unicode hyphens (U+2010) are rendered as "no character"-boxes on my OSMC. 
I see that Kodi itself upgraded to Noto 2.0 about 10 months ago, which includes the hyphen: https://github.com/xbmc/xbmc/commit/951c494e4861d6338076350a5503f7a0183dc91f

Checking the font file in OSMC, it was updated last 6 years ago and is still at version 1.04: https://github.com/osmc/osmc/commit/e56ed09edb036292c9e2a11eb16c2172ac852e0a 

Reasons to update:
* Hyphen character is used often on MusicBrainz (i.e. ending up in many music metadata fields).
* Default font on default skin - so basically out-of-the-box OSMC - should really be able to display something as simple as a hyphen.
* TTF file size did not increase significantly, so that should not be a blocker.

So what I'm saying is, could you please upgrade the Noto font files to a more recent version? :)

**Comment by philippedostert** [comment]
For testing, I copied the new Noto files onto my OSMC (Raspberry Pi 3B) and while it fixes the hyphen U+2010 issue, I see it creates a new problem:
When in a submenu, e.g. any Music menu, the small string displayed below the title on the left now contains a "no character" box. :(

I tracked it down to addons/skin.estuary/xml/Includes.xml:861:
```
<param name="sublabel">$INFO[Container.SortMethod,$LOCALIZE[31022]: ,  ∙  ]$INFO[Container.CurrentItem,, / ]$INFO[Container.NumItems]</param>
```

That's a U+2219 bullet point in there, and it is indeed not part of the newest Noto file taken from Github.

So maybe my request is a bit more complicated than expected...

**Comment by samnazarko** [comment]
Thanks for the report. This tracked is for confirmed issues only.

We simply distribute the Estuary skin as is without modification in this regard. As such, it sounds like the issue is Kodi related and would be best reported on the Kodi forums.

Thanks for your understanding

Sam

**Comment by philippedostert** [comment]
Hi Sam,
thank you for looking at the issue. Let me write down my thoughts and findings, maybe it helps someone down the road.

My issue was "Hyphen character is not rendered correctly", which was caused by the glyph missing in the font file, which is a skin issue, which is Estuary, which is **not** the default skin for OSMC - my bad. I should have done my homework and realized you don't include the Estuary files in your repo.
Instead, I just found some Noto TTFs in the repo and jumped to conclusions.

From what I can tell (I'm no build system expert), you pull in "base" Kodi, which includes the Estuary skin, in `packages/mediacenter-osmc/build.sh`:
```
if [ "$1" == "rbp1" ] || [ "$1" == "rbp2" ] || [ "$1" == "rbp4" ] || [ "$1" == "pc" ] || [ "$1" == "vero2" ] || [ "$1" == "vero3" ]
then
pull_source "https://github.com/xbmc/xbmc/archive/0655c2c71821567e4c21c1c5a508a39ab72f0ef1.tar.gz" "$(pwd)/src"
``` 
That hash is the 18.9 "Leia" release of Kodi, https://github.com/xbmc/xbmc/commit/0655c2c71821567e4c21c1c5a508a39ab72f0ef1

So now my original question becomes "if Kodi/xbmc updated their font files on master branch in June, why did they not flow into the subsequent releases in October?"
And that is a question for Kodi, not for OSMC.

Thanks again!
